deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml

```
@@ -2,6 +2,29 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "toolchain.openshift.dev/v1alpha1",
+          "kind": "CheInstallation",
+          "metadata": {
+            "name": "toolchain-cheinstallation"
+          },
+          "spec": {
+            "cheOperatorSpec": {
+              "namespace": "toolchain-che"
```

I believe it should be `toolchain-workspaces`

```suggestion
              "namespace": "toolchain-workspaces"
```